components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1171,6 +1172,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             }
 
             const workspace = await this.workspaceFactory.createForContext(ctx, user, context, normalizedContextUrl);
+            log.info("Created workspace", { workspace });
```

Comment:
If we want to leave these lines I think they go into `debug` and fill the `LogContext` shape (userId, workspaceId) as a first argument. :lotus_position:

Reply:
Ah, good catch, thanks. I missed this when removing my temporary debug logs. Will remove as well. 🧹